First Commit SOAR Framework v.1.0.1

cs-sujeet-ahirrao opened the pull request:
0811612:
- Add widget Custom Picklist Message v1.1.0 
- Populate the message on the Alert screen stating “Alert is Resolved”. when the status is closed

0811611:
- Modified max Length Constraint field of Alert to 1024 or 2048

0810512
- Remove "Length Constraints" of Source ID of Alert field.

0806454
- Added Custom Picklist Message on Indicator to indicate Enrichment is in process. Added “Classification” picklist which will set/unset in every enrichment.
- Added EmailRep connector for Email Enrichment playbook

0802091
SOAR Framework: Add Activate Users Playbook under 08 - Utilities collection


0801832
categorise the alert as False Positive or True Positive. I need to plot the corresponding charts as well on Dashboard. Perhaps, plotting a Pie around Closure Reason

0801245
SOAR Framework: Add Communication Module Under IR Menu

0801190
Use EmailRep for email enrichment in SOAR SP

0800642
SOAR Framework SP: Add an extra path that is free for the connector

0801189
whois information needs to be conditionally visible

0796900
For Incorrect inputs, The Enrichment playbooks should be handled, currently, playbooks are failing

cs-suyog-jinturkar commented:
1. Modified max Length Constraint field of Alert to 1024 or 2048 - Please mention changed field names 
2. Observed that some fields are changed from 512 to 1024? Why and where are we tracking this change
3. In "alert/mmd/json" and other mmd.json, i can see that some fields are change from -1 to 10485761
e.g.
"displayName": "{{ incidents }}",
"maxlength": 10485761

alertRemainingRespSLA, alertRemainingAckSLA
"minlength": -2147483648,
"maxlength": 10485761

cs-sujeet-ahirrao commented:
1. Below are the modified field names 
Modified max length from 255 to 1024
Source Type
MITRE ATT&CK ID
Attachment Names
Bytes Transferred
Command Line
Computer Name
Decoded Command Line
Destination IP
Destination Port
Device UID
Domain
Email From
Email Recipients (To)
Epoch Time
Event Time
File Hash
File Name
Latest Login
MITRE Technique
Parent Process Command Line
Parent Process ID
Parent Process Name
Process GUID
Process ID
Process Name
Process Tree
Registry Key
Registry Key Value
Reporter
Return Path
Scheduled Task
Sender Domain
Sender Email Address
Service
Source
Source IP
Source Port
Source Process
Source Tool
Target Asset
Target Process
URL
Username



Removed length constraint for the field “Source ID”


Modified max length from 255 to 2048
Email Subject
File Path

2. We have opened the bug for the same "0811611"
3. the M2M fields length constraint cant define by user it is auto change
